vivisect/symboliks/archs/i386.py

```diff
@@ -145,6 +149,17 @@ def setRegObj(self, regidx, obj):
 
         self.effSetVariable(rname, obj)
 
+    def setRegByName(self, name, obj):
```

Comment:
please add a docstr indicating that indicates what this does and any constraints on how it should be used.  i'm not sure i understand myself.

Reply:
:+1: